https://issues.redhat.com/browse/ACM-14988 (#7210)
open-cluster-management/rhacm-docs: release_notes/known_issues_observability.adoc

For more about deprecations and removals, see xref:../release_notes/acm_deprecate_remove.adoc#deprecations-removals-acm[Deprecations and removals for {acm-short}]. 

[#grafana-dashboard-missing]
== Grafana dashboard missing 
//2.12:14639

A Grafana dashboard might fail to load after you run the Grafana instance. Complete the following steps:

. To verify whether a dashboard failed to load, check the logs by running the following command:

+
[source,bash]
----
oc logs observability-grafana-68f8489659-m79rv -c grafana-dashboard-loader -n open-cluster-management-observability
...
E1017 12:55:24.532493 1 dashboard_controller.go:147] dashboard: sample-dashboard could not be created after retrying 40 times
----

> . To fix the dashboard failure, redeploy Grafana by scaling the number of replicas to `0`. The `multicluster-observability-operator` pod automatically scales the number of replicas after you change it to `0`. Run the following command:

Reviewer: The latest edit, makes this a bit more unclear. Just explain in detail a little more what happens when you execute that command is:

1) Kubernetes scales down the number of replicas to 0 as requested in the command
2) The `multicluster-observability-operator` pod sees that the number of replicas are 0, which doesn't match the configured number of replicas, and therefore it scales the number of replicas back up to whatever number is configured in in the multicluster-observability config (2 by default).

```suggestion
. To fix the dashboard failure, redeploy Grafana by scaling the number of replicas to `0`. The `multicluster-observability-operator` pod will automatically scale the deployment back up to the correct number of replicas. Run the following command:
```

> . To fix the dashboard failure, redeploy Grafana by scaling the number of replicas to `0`. The `multicluster-observability-operator` pod automatically scales the number of replicas after you change it to `0`. Run the following command:

Author: @jacobbaungard When we say `multicluster-observability-config` are we referring to the `MultiClusterObservability` resource? I also changed your suggestion to ensure that we use present tense rather than future tense. This is according to the IBM Style Guide. 
```suggestion
. To fix the dashboard failure, redeploy Grafana by scaling the number of replicas to `0`. The `multicluster-observability-operator` pod automatically scales the deployment to the desired number of replicas that is defined in the `MultiClusterObservability` resource. Run the following command:
```

Reviewer: > When we say multicluster-observability-config are we referring to the MultiClusterObservability resource?

The number of replicas are defined in the `MultiClusterObservability` resource, yes. However in most cases I assume defaults are used, and there would be no explicit configuration of this.

I would suggest to say something like "the correct number of replicas" / "desired number of replicas", to ensure people are not confused in case they didn't directly set this in the `MultiClusterObservability` resource.

> . To fix the dashboard failure, redeploy Grafana by scaling the number of replicas to `0`. The `multicluster-observability-operator` automatically scales the number of replicas to the desire amount afterwards. Run the following command:

Author: ```suggestion
. To fix the dashboard failure, redeploy Grafana by scaling the number of replicas to `0`. The `multicluster-observability-operator` pod automatically scales the number of replicas after you change it to `0`. Run the following command:
```
